### ISSUE TITLE
WCAG 2.2: "Focus visible (enhance)" link should be "Focus visible (enhanced)"

### ISSUE THREAD

**Issue by JasonAment** [w3c/wcag#1060]
In ["5.1 New Features in WCAG 2.2"](https://www.w3.org/TR/WCAG22/#new-features-in-wcag-2-2) the link "Focus visible (enhance)" should be "Focus visible (enhanced)"
The following Success Criteria are new in WCAG 2.2:
**Focus visible (enhance)**
The new success criteria may reference new terms that have also been added to the glossary and form part of the normative requirements of the success criteria.

**Comment by alastc** [comment]
Doh! Thanks for that, fixed in master.
